test/e2e_node/services/kubelet.go

```diff
@@ -155,6 +155,7 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		"--serialize-image-pulls", "false",
 		"--pod-manifest-path", manifestPath,
 		"--file-check-frequency", "10s", // Check file frequently so tests won't wait too long
+		"--docker-disable-shared-pid=false",
```

Comment:
If we set it here we don't need the change in `test/e2e_node/jenkins/jenkins-pull.properties`?

Actually, if we don't set it here there are other `properties` files in `test/e2e_node/jenkins/` need to be changed.

Reply:
Ok, thanks. Based on this advice (and because it made the e2e node test pass) I've removed the jenkins config and set the flag here.